travis/publish.sh

```diff
@@ -71,16 +85,17 @@ release_version() {
     echo "${TRAVIS_TAG}" | sed 's/^release-//'
 }
 
-safe_checkout_master() {
-  # We need to be on a branch for release:perform to be able to create commits, and we want that branch to be master.
-  # But we also want to make sure that we build and release exactly the tagged version, so we verify that the remote
-  # master is where our tag is.
-  git checkout -B master
-  git fetch origin master:origin/master
-  commit_local_master="$(git show --pretty='format:%H' master)"
-  commit_remote_master="$(git show --pretty='format:%H' origin/master)"
-  if [ "$commit_local_master" != "$commit_remote_master" ]; then
-    echo "Master on remote 'origin' has commits since the version under release, aborting"
+safe_checkout_remote_branch() {
+  # We need to be on a branch for release:perform to be able to create commits,
+  # and we want that branch to be master or v0.0.0. which has been checked before.
+  # But we also want to make sure that we build and release exactly the tagged version,
+  # so we verify that the remote branch is where our tag is.
+  git checkout -B v`release_version | sed 's/-RC[[:digit:]]\+//'`
```

Comment:
This has to be changed. 

It should distinguish between master and version branch. Please update it with ot-java repo.

Reply:
Then it can be merged/released

Reply:
Assume you meant to just copy it from ot-java repo - if so, done 👍